shared/src/business/entities/EntityConstants.ts

```diff
@@ -6,6 +6,10 @@ import courtIssuedEventCodesJson from '../../tools/courtIssuedEventCodes.json';
 import externalFilingEventsJson from '../../tools/externalFilingEvents.json';
 import internalFilingEventsJson from '../../tools/internalFilingEvents.json';
 
+export const KEYS = {
+  DOCKET_RECORD_TABLE_SORT: 'DOCKET_RECORD_TABLE_SORT',
+};
```

Comment:
Can you explain why we use `KEYS` and `ROOT`? It seems like we are using this as a way to dynamically set state based on keys, but this worries me: working with Cerebral can already be opaque enough (always interacting with a global state object), and this approach seems to "hide" state (it cannot be seen in `state.ts`, and it isn't immediately obvious where this state is defined/typed).

To be clear: I'm not saying this approach is wrong or bad, I just want to understand it better!

In any case, if we do use ROOT/KEY, I think we should make these names more descriptive. A root into what? What sort of key? Etc. These terms need context.